frontend/client/components/RFPs/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Row, Col } from 'antd';
+import { fetchRfps } from 'modules/rfps/actions';
+import { AppState } from 'store/reducers';
+import { RFP } from 'types';
+import { RFP_STATUS } from 'api/constants';
+import Loader from 'components/Loader';
+import Placeholder from 'components/Placeholder';
+import RFPCard from './RFPCard';
+import ZCFLogo from 'static/images/zcf.svg';
+import './index.less';
+
+interface StateProps {
+  rfps: AppState['rfps']['rfps'];
+  isFetchingRfps: AppState['rfps']['isFetchingRfps'];
+  fetchRfpsError: AppState['rfps']['fetchRfpsError'];
+}
+
+interface DispatchProps {
+  fetchRfps: typeof fetchRfps;
+}
+
+type Props = StateProps & DispatchProps;
+
+class RFPs extends React.Component<Props> {
+  componentDidMount() {
+    this.props.fetchRfps();
+  }
+
+  render() {
+    const { rfps, isFetchingRfps, fetchRfpsError } = this.props;
+
+    let rfpsEl;
+    if (fetchRfpsError) {
+      rfpsEl = (
+        <div className="RFPs-error">
+          <Placeholder
+            title="Something went wrong"
+            subtitle="We had an issue fetching requests, try again later"
+          />
+        </div>
+      );
+    }
+    else if (!isFetchingRfps) {
+      rfpsEl = (
+        <div className="RFPs-loading">
+          <Loader size="large" />
+        </div>
+      );
+    }
+    else {
+      const live = rfps.filter(rfp => rfp.status === RFP_STATUS.LIVE);
+      const closed = rfps.filter(rfp => rfp.status === RFP_STATUS.CLOSED);
+      rfpsEl = <>
+        {this.renderRfpsList('Open Requests', live)}
+        {closed.length && this.renderRfpsList('Closed Requests', closed)}
```

Comment:
`!!closed.length` to avoid rendering '0'

Reply:
A mighty fine catch. Fixed!